superset-frontend/src/dashboard/components/nativeFilters/FilterConfigForm.tsx

```diff
@@ -0,0 +1,223 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { styled, t } from '@superset-ui/core';
+import { FormInstance } from 'antd/lib/form';
+import React, { useCallback, useState } from 'react';
+import {
+  Checkbox,
+  Form,
+  Input,
+  Radio,
+  Typography,
+} from 'src/common/components';
+import SupersetResourceSelect, {
+  Value,
+} from 'src/components/SupersetResourceSelect';
+import { addDangerToast } from 'src/messageToasts/actions';
+import { ClientErrorObject } from 'src/utils/getClientErrorObject';
+import { ColumnSelect } from './ColumnSelect';
+import ScopingTree from './ScopingTree';
+import { Filter, NativeFiltersForm, Scoping } from './types';
+
+type DatasetSelectValue = {
+  value: number;
+  label: string;
+};
+
+const datasetToSelectOption = (item: any): DatasetSelectValue => ({
+  value: item.id,
+  label: item.table_name,
+});
+
+const ScopingTreeNote = styled.div`
+  margin-top: -20px;
+  margin-bottom: 10px;
```

Comment:
```suggestion
  margin-bottom: 10px;
```
Not sure if 2 or 3 gridUnits would look better here... but let's round to the best gridUnit ◰